website/docs/r/compute_region_autoscaler.html.markdown

```diff
@@ -0,0 +1,155 @@
+---
+layout: "google"
+page_title: "Google: google_compute_region_autoscaler"
+sidebar_current: "docs-google-compute-region-autoscaler"
+description: |-
+  Manages a Regional Autoscaler within GCE.
+---
+
+# google\_compute\_region\_autoscaler
+
+A Compute Engine Regional Autoscaler automatically adds or removes virtual machines from
+a managed instance group based on increases or decreases in load. This allows
+your applications to gracefully handle increases in traffic and reduces cost
+when the need for resources is lower. You just define the autoscaling policy and
+the autoscaler performs automatic scaling based on the measured load. For more
+information, see [the official
+documentation](https://cloud.google.com/compute/docs/autoscaler/) and
+[API](https://cloud.google.com/compute/docs/reference/latest/regionAutoscalers)
+
+
+## Example Usage
+
+```hcl
+resource "google_compute_instance_template" "foobar" {
+  name           = "foobar"
+  machine_type   = "n1-standard-1"
+  can_ip_forward = false
+
+  tags = ["foo", "bar"]
+
+  disk {
+    source_image = "debian-cloud/debian-8"
+  }
+
+  network_interface {
+    network = "default"
+  }
+
+  metadata {
+    foo = "bar"
+  }
+
+  service_account {
+    scopes = ["userinfo-email", "compute-ro", "storage-ro"]
+  }
+}
+
+resource "google_compute_target_pool" "foobar" {
+  name = "foobar"
+}
+
+resource "google_compute_region_instance_group_manager" "foobar" {
+  name = "foobar"
```

Comment:
Can you line up the `=`s here and below?

Reply:
Done